tests/rustdoc-json/keyword.rs

```diff
@@ -15,6 +15,6 @@ pub mod foo {}
 
 //@ !has "$.index[*][?(@.name=='hello')]"
```

Comment:
This should be changed to check “break” instead of “hello”.

Reply:
ok